arguments/SPIR-V.argdown

```diff
@@ -56,8 +60,11 @@ functionality present in core SPIR-V, like the instructions for ray tracing.
 (3) That will effectively lead to the forking SPIR-V
   -> [SPIR-V]
 
-<Nobody wants yet another shading language>: we already have a diversity of shading languages,
-as well as huge shader code bases. Adding support for another one is something nobody is excited about.
+<No Gatekeeping solely by the browser vendors>:
```

Comment:
WebGPU is a standard within W3C. You can't bring an argument that WebGPU API needs to be developed by something other than the group. This point should be removed or changed radically

Reply:
> JS adopted CSS selectors, JSON, ISO dates, regexp that are all specified by different groups.
HTML refers to multiple external specs. I must not understand your point.

I understand better what you mean now. Still, saying that it's better to have stuff borrowed from a different group just because it's not in W3C is not a good argument to make.

Also, WebGL2 situation is not gatekeeping, it's just lack of engineering resources in some companies.

Reply:
I used to work at Apple, everything management supports gets budget. No budget at Apple is a corporate way of saying we don't care or we don't want it.

Reply:
Yes, corporations can change priorities (like Apple changing course from OpenGL to Metal), of course that happens.
If they don't want WebGPU *at some point* there is not much we can do. It's not a problem with WGSL specifically.